File: Android/EasyHero/app/src/main/java/com/example/easyhero/home/Hero.kt
package com.example.easyhero.home

data class Hero(
    val id : Int,
    val name: String,
    val realName: String,
    val imageUrl: String,
    val powerstats: Powerstats
)

data class Powerstats(
    val intelligence: String,
    val strength: String,
    val speed: String,
    val durability: String,
    val power: String,
    val combat: String
)

File: IOS/EasyHero/EasyHero/Domain/Hero.swift
//
//  Hero.swift
//  EasyHero
//
//  Created by Alumno on 12/07/25.
//

struct Hero: Identifiable, Equatable {
    let id: Int
    let name: String
    let fullName: String
    let image: String
    let stats: HeroStats
}

struct HeroStats: Equatable {
    let intelligence: String
    let strength: String
    let speed: String
    let durability: String
    let power: String
    let combat: String
}
